packages/storybook8/stories/Concepts/Theming/Docs.mdx

```diff
@@ -0,0 +1,113 @@
+import { Meta, Controls, Primary, Source } from '@storybook/addon-docs';
+import { DarkControlBar } from './DarkControlBar.story';
+import { DefaultTheme } from './DefaultTheme.story';
+import { ThemedButton } from './ThemedButton.story';
+
+import DarkControlBarText from '!!raw-loader!./snippets/DarkControlBar.snippet.tsx';
+import DefaultThemeSnippetText from '!!raw-loader!./snippets/DefaultTheme.snippet.tsx';
+import ThemedButtonText from '!!raw-loader!./snippets/ThemedButton.snippet.tsx';
+
+<Meta title="Concepts/Theming" />
+
+# Theming
+
+ACS UI Library uses components and icons from both Fluent UI React [v8](https://aka.ms/fluentui#/controls/web)
+and [v9](https://react.fluentui.dev/). The `FluentThemeProvider`
+is a wrapping component that allows you to theme all ACS UI Library components consistently.
+
+## Usage
+
+ACS UI Library components are themed with a light theme by default as shown by the `ControlBar` and buttons below.
+
+<Source code={DefaultThemeSnippetText} />
+<DefaultTheme />
+
+To theme this ControlBar, import `FluentThemeProvider` from `@azure/communication-react` and wrap it around the `ControlBar`.
+Then import `darkTheme` from `@azure/communication-react` and assign it to the `fluentTheme` property of `FluentThemeProvider`.
+`darkTheme` is a [Theme](https://aka.ms/fluentui#/controls/web/references/theme) object from Fluent UI.
+
+<Source code={DarkControlBarText} />
+<DarkControlBar />
+
+## FluentThemeProvider Props
```

Comment:
I assume we're still trying to figure out props?

Reply:
Yes this is an ongoing issue we need to figure out